js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetNode2025.tsx

```diff
@@ -144,6 +149,89 @@ export const AssetNodeWithLiveData = ({
   );
 };
 
+export const AssetNodeAutomationRow = ({
+  definition,
+  liveData,
+}: {
+  definition: AssetNodeFragment;
+  liveData: LiveDataForNode | undefined;
+}) => {
+  const hasAutomationCondition = !!definition.automationCondition;
+  const sensors = definition.targetingInstigators.filter(
+    (instigator) => instigator.__typename === 'Sensor',
+  );
+  const hasSensors = !!sensors.length;
```

Comment:
Could do something like this for sensor/sechedule:

```
const firstSensor = sensors[0];
```

Then use that instead of `hasSensors`, so that you don't need to do `!` assertions on the array element.